packages/tpetra/core/src/Tpetra_CrsGraph_def.hpp

```diff
@@ -5855,6 +5881,13 @@ namespace Tpetra {
        "exportLIDs.size() = " << numExportLIDs << " != numPacketsPerLID.size()"
        " = " << numPacketsPerLID.size () << ".");
 
+    // FENCE REVIEW - NOT TESTED
+    //   Testing: This code is NOT exercised by unit tests.
+    //   GTX960:  N/A
+    //   White:   N/A
+    //   Plan:    Track if graph is safe and fence based on that.
+    //   Notes:   Need fence for getLocalViewRawConst and getGlobalViewRawConst
+
```

Comment:
This case is exercised when packAndPrepare is called on a RowGraph, rather than on a CrsGraph.   I assume this use case is needed for Ifpack2; is that correct, @trilinos/ifpack2?

In this case, packAndPrepare calls DistObject's pack() method, which calls packFillActive(), which has these fences.
Method packFillActive() is needed only because of the old Teuchos::ArrayView<> interface to the pack() method; otherwise, I think packFillActiveNew() could be called.

If we need this capability, we should add a unit test for it.
Should we change the interface to DistObject::pack() to use Kokkos::Views instead of Teuchos::ArrayView()?  Or would that create too much work in Ifpack2?